### PR TITLE
ref #342: Add permission for all languages and portals

### PR DESCRIPTION
**Pull request by umulmrum**
| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#342
| License       | MIT

### CODE REVIEW

File: src/DistributionBundle/Bootstrap/InitialBackendUserCreator.php [hunk 1]
```diff
@@ -159,11 +159,47 @@ private function addUser(string $userId, string $name, string $password): void
...
+                'entry_sort' => ++$position,
+            ]);
+        }
+    }
+
+    private function getAllLanguages(): array
+    {
+        $languages = [];
+        $languageList = \TdbCmsLanguageList::GetList();
+        while (false !== $languageObject = $languageList->Next()) {
```
Comment (UlrichKu) [inline src/DistributionBundle/Bootstrap/InitialBackendUserCreator.php]: Why not loop these above directly?

(As these methods also do not get all languages but all ids.)

Comment (umulmrum) [reply src/DistributionBundle/Bootstrap/InitialBackendUserCreator.php]: Yes, that was overly complex - changed.

File: src/DistributionBundle/Bootstrap/InitialBackendUserCreator.php [hunk 2]
```diff
@@ -159,11 +159,34 @@ private function addUser(string $userId, string $name, string $password): void
             'show_as_rights_template' => '1',
         ]);
 
-        $this->databaseConnection->insert('cms_user_cms_language_mlt', [
-            'source_id' => $userId,
-            'target_id' => $languageEn->id,
-            'entry_sort' => 1,
-        ]);
+        $this->addLanguageLinks($userId);
```
Comment (UlrichKu) [inline src/DistributionBundle/Bootstrap/InitialBackendUserCreator.php]: connectUserWithPortals? (or similar) "Links" is a bit arbitrary

Comment (umulmrum) [reply src/DistributionBundle/Bootstrap/InitialBackendUserCreator.php]: If it makes you happy .. ;-)

**Pull request merged**